Gemfile

```diff
@@ -30,6 +30,9 @@ gem 'coffee-rails', '~> 4.0.0'
 # Use jquery as the JavaScript library
 gem 'jquery-rails'
 
+group :development, :test do
+  gem 'pry-rails'
+end
```

Comment:
Great add!  If you guys don't know how to use pry look it up.  Great debugging tool.  Or perhaps pair with kirubel for a few minutes to let him teach you.